panther_analysis_tool/schema_regexs.py

```diff
@@ -31,6 +31,9 @@
     r"Bitwarden\.Events|"
     r"Box\.Event|"
     r"CarbonBlack\.Audit|"
+    r"CarbonBlack\.AlertV2|"
+    r"CarbonBlack\.EndpointEvent|"
+    r"CarbonBlack\.WatchlistHit|"
```

Comment:
was this added manually? my understanding is we have automation that generates this file. did it not run somehow?

Reply:
Ah, yea -- `# Autogenerated using the pat_regex_generator dev tool`. I'll run that instead.